tests/screenshotter/scripts/on-before-ready.js

```diff
@@ -18,6 +20,15 @@ module.exports = async function( page ) {
 	}
 
 	page
+		.on( 'request', async ( request ) => {
+			const req = request.url();
+			const host = url.parse( config.url_origin, true ).host;
+			const reqUrl = url.parse( req, true ).host;
+			if ( 'localhost' === reqUrl ) {
+				req.replace( reqUrl, host );
+				return req;
```

Comment:
1. Fix the names to be accurate:  `req` is actually the `requestUrl` and `reqUrl` is in fact the `requestHost`. `host` is the `originHost`
2. It's doesn't work because the host is not including the port.

Reply:
1. Fixed
2. Work well, host  return the host name and port